### PR TITLE
fixes for my workflow

### PR DESCRIPTION
**Pull request by ndp**
Hey thanks! It worked.

Take or leave these as you see fit. I tried to make only minor changes... I'm going to also try importing to Github Issues now, so not doing much more for the moment, but I have a half dozen projects to move over.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,11 +4,15 @@
...
+  },
   "dependencies": {
     "@linear/sdk": "^30.0.0",
     "chalk": "^5.3.0",
     "csv-parse": "^5.5.6",
     "dotenv": "^16.4.5",
     "inquirer": "^11.0.2"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```
Comment (ndp) [inline package.json:17]: yarn did this for us.

File: src/csv/parse.mjs [hunk 2]
```diff
@@ -85,14 +85,18 @@ async function parseCSV() {
...
   const filePath = path.join(assetsDir, selectedDirectory, csvFilename);
   
+  return parseCSVFile(csvFilename, filePath)
+}
+
+export async function parseCSVFile(csvFilename, filePath) {
   if (!await fs.access(filePath).then(() => true).catch(() => false)) {
     console.log(`CSV file not found: ${csvFilename}`);
-    return;
+    process.exit(1);
```
Comment (ndp) [inline src/csv/parse.mjs:94]: The `CSV file not found` message was getting lost in the various extra stack traces output.

File: src/import.mjs [hunk 3]
```diff
@@ -38,7 +38,10 @@ const logger = setupLogger(teamName);
 logger.enable();
 
 // PROMPTS
-const { releaseStories, pivotalStories, labels, csvFilename } = await parseCSV();
+const { releaseStories, pivotalStories, labels, csvFilename } = 
+  process.argv[2]
```
Comment (ndp) [inline src/import.mjs:42]: I don't love the "2" here, but this takes into account the "node" and "import.mjs" pieces.

File: src/csv/parse.mjs (hunk 2 above)
Comment (nverges) [reply src/csv/parse.mjs:94]: Good call.

**Review by nverges — APPROVED** [review]

**Pull request merged**